src/hooks/useAADAuth.ts

```diff
@@ -81,9 +83,16 @@ export function useAADAuth(): ReturnType {
           authority: `https://login.microsoftonline.com/${tenantId}`,
           scopes: ["https://management.azure.com//.default"],
         }),
-      ]).then(([graphTokenResponse, armTokenResponse]) => {
+        msal.acquireTokenSilent({
+          // There is a bug in MSALv1 that requires us to refresh the token. Their internal cache is not respecting authority
+          forceRefresh: true,
+          // authority: `https://login.microsoftonline.com/${tenantId}`,
+          scopes: ["https://stfaul-sql.documents.azure.com/.default"],
```

Comment:
for testing

Reply:
This has to be done different. We shouldn't deploy this with a hard coded value. We should move this to another place where it can be loaded after the database account is loaded